docs/editing.md

```diff
@@ -23,9 +23,19 @@ The EiC assigns an editor (or a volunteering editor self-assigns) with the comma
 .. note:: If a paper is submitted without a recommended editor, it will show up in the weekly digest email under the category ‘Papers currently without an editor.’ Please review this weekly email and volunteer to edit papers that look to be in your domain. If you choose to be an editor in the issue thread type the command ``@whedon assign @yourhandle as editor``
 ```
 
+### How papers are assigned to editors
+
+By default, unless an editor volunteers, one of the Editors-in-chief (EiCs) will attempt to assign an incoming paper to the most suitable handling editor. While EiCs will make every effort to match a submission with the most appropriate editor, there are a number of situations where an EiC may assign a paper to an editor that doesn't fit entirely within the editor's research domains:
+
+- If there's no obvious fit to _any_ of the JOSS editors
+- If the most suitable editor is already handling a large number of papers
+- If the chosen editor has a lighter editorial load than other editors
+
+Normally an EiC will ask one or more editors to edit a submission (e.g. `@editor1, @editor 2 - would one of you be willing to edit this submission for JOSS`). If the editor doesn't respond withing a 48 hours, the EiC may assign the paper to the editor regardless.
```

Comment:
In most cases, an AEiC will ask one or more editors to edit a submission (e.g. `@editor1, @editor 2 - would one of you be willing to edit this submission for JOSS`). If the editor doesn't respond within a 48 hours, the AEiC may assign the paper to the editor regardless.